src/runtime/pkg/oci/utils.go

```diff
@@ -770,6 +770,12 @@ func addHypervisporNetworkOverrides(ocispec specs.Spec, sbConfig *vc.SandboxConf
 		return err
 	}
 
+	if err := newAnnotationConfiguration(ocispec, vcAnnotations.QueueNums).setUint(func(queueNums uint64) {
+       		sbConfig.HypervisorConfig.QueueNums = uint32(queueNums)
+     	}); err != nil {
+       		return err
+    	}
+
```

Comment:
Maybe move this a few lines up so that `RxRateLimiterRate` and `TxRateLimiterMaxRate` stay side by side ?

Reply:
Thanks for your suggestions, I will move it.